docs/README.md

```diff
@@ -365,7 +365,7 @@ For Transformers, the doctests are run on a daily basis via GitHub Actions as ca
 seen [here](https://github.com/huggingface/transformers/actions/workflows/doctests.yml).
 
 To include your example in the daily doctests, you need to add the filename that
-contains the example docstring to the [documentation_tests.txt](../utils/documentation_tests.txt).
+contains the example docstring to the [documentation_tests.txt](https://github.com/huggingface/transformers/blob/main/utils/slow_documentation_tests.txt).
```

Comment:
We can get rid of this line completely, `documentation_tests.txt` has been removed and the user doesn't need to add the tests anywhere

Reply:
It's my pleasure! Should I create another PR and delete the line?

Reply:
Nope - you can keep this one, make the change and update the PR description